package.json

```diff
@@ -11,7 +11,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov",
+    "test": "node node_modules/lab/bin/lab -a code -t 100 -L",
```

Comment:
FWIWI this helps only on windows, because it doesn't understand shebangs, but I don't think we care.